core-feature-pack/src/main/resources/content/docs/contrib/scripts/init.d/wildfly-init-debian.sh

```diff
@@ -16,7 +16,7 @@
 # Description:          Provide WildFly startup/shutdown script
 ### END INIT INFO
 
-NAME=wildfly
+NAME=$(basename ${0})
```

Comment:
Note to self: I don't think this is directly part of WFCORE-1604 but it makes this script consistent with wildfly-init-redhat.sh which seems like a good thing.